CMakeLists.txt

```diff
@@ -18,7 +18,16 @@ include(Sanitizers)
 include(RealmCore)
 set(REALM_CORE_VERSION "1.5.0" CACHE STRING "")
 use_realm_core(${REALM_CORE_VERSION})
-build_realm_sync(${REALM_SYNC_PREFIX})
+
+option(REALM_ENABLE_SYNC "Build realm object store with sync extensions" OFF)
```

Comment:
Unless this is part of your plan to work towards using prebuilt versions of sync in the object store, in which case maybe it's fine to leave this.